frontend/src/Utilities/dataUtilities.ts

```diff
@@ -0,0 +1,432 @@
+  {
+    title: "Name",
+    field: "name",
+  },
+  {
+    title: "Balance",
+    field: "balance",
+  },
+  {
+    title: "Account #",
+    field: "account",
+  },
+  {
+    title: "Routing #",
+    field: "routing",
+  },
+];
+
+export const transactionsCategories: Array<Categories> = [
+  {
+    title: "Name",
+    field: "name",
+  },
+  {
+    title: "Amount",
+    field: "amount",
+  },
+  {
+    title: "Date",
+    field: "date",
+  },
+];
+
+export const identityCategories: Array<Categories> = [
+  {
+    title: "Names",
+    field: "names",
+  },
+  {
+    title: "Emails",
+    field: "emails",
+  },
+  {
+    title: "Phone numbers",
+    field: "phoneNumbers",
+  },
+  {
+    title: "Addresses",
+    field: "addresses",
+  },
+];
+
+export const balanceCategories: Array<Categories> = [
+  {
+    title: "Name",
+    field: "name",
+  },
+  {
+    title: "Balance",
+    field: "balance",
+  },
+  {
+    title: "Subtype",
+    field: "subtype",
+  },
+  {
+    title: "Mask",
+    field: "mask",
+  },
+];
+
+export const investmentsCategories: Array<Categories> = [
+  {
+    title: "Account Mask",
+    field: "mask",
+  },
+  {
+    title: "Name",
+    field: "name",
+  },
+  {
+    title: "Quantity",
+    field: "quantity",
+  },
+  {
+    title: "Close Price",
+    field: "price",
+  },
+  {
+    title: "Value",
+    field: "value",
+  },
+];
+
+export const liabilitiesCategories: Array<Categories> = [
+  {
+    title: "Name",
+    field: "name",
+  },
+  {
+    title: "Type",
+    field: "type",
+  },
+  {
+    title: "Last Payment Date",
+    field: "date",
+  },
+  {
+    title: "Last Payment Amount",
+    field: "amount",
+  },
+];
+
+export const itemCategories: Array<Categories> = [
+  {
+    title: "Institution Name",
+    field: "name",
+  },
+  {
+    title: "Billed Products",
+    field: "billed",
+  },
+  {
+    title: "Available Products",
+    field: "available",
+  },
+];
+
+export const accountsCategories: Array<Categories> = [
+  {
+    title: "Name",
+    field: "name",
+  },
+  {
+    title: "Balance",
+    field: "balance",
+  },
+  {
+    title: "Subtype",
+    field: "subtype",
+  },
+  {
+    title: "Mask",
+    field: "mask",
+  },
+];
+
+export const transformAuthData = (data: AuthGetResponse) => {
+  return data.numbers!.ach!.map((achNumbers) => {
+    const account = data.accounts!.filter((a) => {
+      return a.account_id === achNumbers.account_id;
+    })[0];
+    const obj: DataItem = {
+      name: account.name,
+      balance:
+        formatter.format(account.balances.available!) ||
+        formatter.format(account.balances.current!),
+      account: achNumbers.account!,
+      routing: achNumbers.routing!,
+    };
+    return obj;
+  });
+};
+
+export const transformTransactionsData = (
+  data: TransactionsGetResponse
+): Array<DataItem> => {
+  return data.transactions!.map((t) => {
+    const item: DataItem = {
+      name: t.name!,
+      amount: formatter.format(t.amount!),
+      date: t.date!,
```

Comment:
lets check if a newer beta of `plaid-node` has been released that fixes the issue with needing all these `!` because of the optional types